src/material/datepicker/calendar-header.html

```diff
@@ -2,11 +2,12 @@
   <div class="mat-calendar-controls">
     <button mat-button type="button" class="mat-calendar-period-button"
             (click)="currentPeriodClicked()" [attr.aria-label]="periodButtonLabel"
-            [attr.aria-describedby]="_buttonDescriptionId"
-            aria-live="polite">
-      <span [attr.id]="_buttonDescriptionId">{{periodButtonText}}</span>
+            [attr.aria-description]="periodButtonDescription" aria-live="polite">
+      <span [attr.id]="_buttonDescriptionId" aria-hidden="true">
```

Comment:
Is anything referring to this ID now? If not we can just remove it

Reply:
cool, I deleted it since it's not used anymore.